lit_parrot/config.py

```diff
@@ -50,21 +50,21 @@ def from_name(cls, name: str, **kwargs: Any) -> Self:
 ####################
 pythia = {
     # https://huggingface.co/EleutherAI/pythia-70m/blob/main/config.json
-    "pythia-70m": dict(block_size=2048, n_layer=6, n_embd=512, n_head=8, padding_multiple=128),
+    "pythia-70m": dict(block_size=2048, vocab_size=50304, n_layer=6, n_embd=512, n_head=8, padding_multiple=128),
```

Comment:
This is not quite correct. This number you added is for the `padded_vocab_size`. In our case, it is dynamically generated by finding the closest multiple of `padding_multiple` of the `vocab_size`.

GPT-neox chooses a different padding_multiple based on the distributed configuration. It seems like this is different for the 6.9b and 12b variants.

So this should be reverted and instead we need to set either the `padded_vocab_size` or set the correct `padding_multiple` for those two configs.